app/controllers/application_controller.rb

```diff
@@ -4,11 +4,22 @@ class ApplicationController < ActionController::API
   include Pundit
 
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :set_locale, if: :user_signed_in?
```

Comment:
Warning, all actions set_locale before execute. We only need to translate rent email